.markdownlint.yaml

```diff
@@ -0,0 +1,6 @@
+default: true
+MD013: false
+MD024:
+  siblings_only: true
+MD033: false
+MD041: false
```

Comment:
Adding MD046 is good if you use `admonition`.
https://github.com/markdownlint/markdownlint/issues/215

Reply:
I'm adding it.

Reply:
f8405941a353c96e23eeef2a2e3545b705ad3891